exe/PowershellShim-Helper.ps1

```diff
@@ -6,7 +6,12 @@
 #
 #  > $Json | ConvertFrom-json -Type PSObject
 
-Add-Type -AssemblyName System.ServiceModel.Web, System.Runtime.Serialization
+try {
+  Add-Type -AssemblyName System.ServiceModel.Web, System.Runtime.Serialization
```

Comment:
I would be tempted to add `-ErrorAction 'Stop'` at the end so that you _know_ any errors will be terminal

Reply:
To the Add-Type call? Ok, I'll do that when I update the phrasing.